node/src/options.rs

```diff
@@ -37,18 +37,12 @@ pub struct Options {
     #[clap(short, long = "datadir")]
     pub data_dir: Option<PathBuf>,
 
-    /// The path to the config file.
-    #[clap(short, long = "conf")]
-    config_path: Option<PathBuf>,
```

Comment:
Bitcoin accepts such command line option, but I also think that removing it makes things easier.

Reply:
Yeah, I don't think this is useful... I'm not convinced. It just complicates things for no good reason. Unless a usecase appears, I'd like to keep this off.